mne/dipole.py

```diff
@@ -375,6 +375,12 @@ def __init__(self, info, data, times, nave, aspect_kind, first, last,
         self.data = data
         self.verbose = verbose
 
+    def __repr__(self):  # noqa: D105
+        s = "n_times : %s" % len(self.times)
+        s += ", tmin : %s" % np.min(self.times)
+        s += ", tmax : %s" % np.max(self.times)
+        return "<DipoleFixed  |  %s>" % s
```

Comment:
Some questions:
1. How do I create a new PR (for issue 4149) without joining it?
2. Do I need to create a test for this or just test with print on my machine? See it, I already reproduce it:
```
>>> dipoles, residual = rap_music(evoked, fwd, noise_cov, n_dipoles=2, return_residual=True, verbose=False)
>>> print dipoles[0]
<DipoleFixed  |  n_times : 421, tmin : -0.199795213158, tmax : 0.499488032896>
```